resources/images/controller/images_external_test.go

```diff
@@ -463,7 +463,7 @@ func TestSoftwareImagesControllerNewImage(t *testing.T) {
 
 		model.On(
 			"CreateImage",
-			mock.AnythingOfType("*os.File"),
+			mock.Anything,
```

Comment:
I think it should be possible to use `mock.MatchedBy()` here, like this:

```
model.On(
    "CreateImage",
    mock.MatchedBy(func(ir interface{}) bool {
        _, ok := ir.(io.Reader)
        return ok
     }),
    ...
    )
```

care to try?